packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -239,7 +239,7 @@ type Params = {
  * //  key being used in dbAccessor in src/functions/auth.ts 👇
  * const getCurrentUser = async (session: DbAuthSession<User['id']>)
  */
-export interface DbAuthSession<TIdType = unknown> {
+export interface DbAuthSession<TIdType = any> {
```

Comment:
@Tobbe had to set this to any again, because it complains about currentUser function in strict mode otherwise.

Reply:
Next time I see this I'll get an itch to try to switch it to `unknown` again... How did you notice it was complaining?

Reply:
Just a tsc error in the projects graphql function. 

It happens because findUnique (when trying to find the user) can return null I think